lib/ui/text.dart

```diff
@@ -80,15 +81,15 @@ class FontWeight {
   ///
   /// Values for `t` are usually obtained from an [Animation<double>], such as
```

Comment:
Nitpick: \`t\` should be `[t]`

Reply:
It's one of those quirks in the Flutter's style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements